Header header

std_msgs/Float64 distance_traveled
std_msgs/Float64 energy_spent
std_msgs/Float64 mean_speed
float64[] closest_approach
std_msgs/Int32 score
std_msgs/Float64 mean_cpu_usage
std_msgs/Int32 penalties
